src/REL/server.py

```diff
@@ -146,25 +176,41 @@ def generate_response(self, text, spans):
     from http.server import HTTPServer
 
     from REL.entity_disambiguation import EntityDisambiguation
-    from REL.ner import load_flair_ner
+    from REL.ner.flair_wrapper import load_flair_ner
+    from REL.ner.bert_wrapper import load_bert_ner
 
     p = argparse.ArgumentParser()
-    p.add_argument("base_url")
-    p.add_argument("wiki_version")
+    p.add_argument("--base_url", default=os.path.abspath(os.path.dirname(__file__) + "/../../data/"))
+    p.add_argument("--wiki_version", default="wiki_2019")
     p.add_argument("--ed-model", default="ed-wiki-2019")
     p.add_argument("--ner-model", default="ner-fast")
     p.add_argument("--bind", "-b", metavar="ADDRESS", default="0.0.0.0")
     p.add_argument("--port", "-p", default=5555, type=int)
+    p.add_argument("--use_bert_large_cased", help = "use Bert large cased rather than Flair", action="store_true")
+    p.add_argument("--use_bert_base_cased", help = "use Bert base cased rather than Flair", action="store_true")
+    p.add_argument("--use_bert_large_uncased", help = "use Bert large uncased rather than Flair", action="store_true")
+    p.add_argument("--use_bert_base_uncased", help = "use Bert base uncased rather than Flair", action="store_true")
+    p.add_argument("--use_bert_multilingual", help = "use Bert multilingual rather than Flair", action="store_true")
+    p.add_argument("--process_sentences", help = "process sentences rather than documents", action="store_true")
+    p.add_argument("--split_docs_value", help = "threshold number of tokens to split document")
+
     args = p.parse_args()
 
-    ner_model = load_flair_ner(args.ner_model)
+    tagger_ner, tagger_ner_name = set_tagger_ner(args.use_bert_base_cased, args.use_bert_base_uncased, args.use_bert_large_cased, args.use_bert_large_uncased, args.use_bert_multilingual)
+
+    split_docs_value = 0
+    if args.split_docs_value:
+        split_docs_value = int(args.split_docs_value)
```

Comment:
Set defaults in argparse.
```suggestion
    split_docs_value = args.split_docs_value
```

Reply:
Adopted: add defaults for arguments